testing/refinerycms-testing.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.files             = `git ls-files`.split("\n")
 
   s.add_dependency 'refinerycms-core',        version
-  s.add_dependency 'database_cleaner',        '~> 1.6'
+  s.add_dependency 'database_cleaner',        '>= 1.6'
```

Comment:
do we even need this anymore? https://stackoverflow.com/a/49248170

Reply:
Probably not but i will not handle this case in this PR